src/components/backend-ai-metadata-store.ts

```diff
@@ -86,6 +91,11 @@ export default class BackendAIMetadataStore extends BackendAIPage {
           }
         }
       }
+    ).then(
+      () => {
+        const event: CustomEvent = new CustomEvent('backend-ai-metadata-device-loaded', {'detail': ''});
```

Comment:
I cannot find eventListener of this. Is it intended?

Reply:
Yes it is a part of next PR.